rust/rope/src/engine.rs

```diff
@@ -110,7 +119,7 @@ impl Engine {
 
     /// Get text of head revision.
     pub fn get_head(&self) -> Rope {
-        self.rev_content_for_index(self.revs.len() - 1)
+        self.text.clone()
```

Comment:
Okay done, at the moment every non-test reference to `get_head` now immediately clones it, but that could change in the future.